CONTRIBUTING.md

```diff
@@ -8,6 +8,48 @@ When you submit a pull request, a CLA bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+# Types of Contributions
+
+## Submit a pull request that aligns with our goals and priorities

```

Comment:
These should probably be `##` and `###` respectively, so as to be placed under the `Contributing` \<h1\>.